src/svg/HomeSVG.js

```diff
@@ -10,6 +11,7 @@ Animated.addWhitelistedNativeProps({
 
 const HomeSVG = ({color, size}: SVGProps) => {
   return (
+    <View testID="FeedSVG">
```

Comment:
Some testIDs are written in camelCase, others are written in lisp-case. Best to pick a single format and stick with it for consistency